src/utils/adapters.js

```diff
@@ -54,6 +54,7 @@ export function adaptTransactions(transactions) {
 }
 
 export function adaptContracts(contracts) {
+  console.log('contracts', contracts)
```

Comment:
There are some `console.logs`, could you please check it and remove them?

Reply:
ooops, removed